docs/settings.rst

```diff
@@ -21,26 +21,36 @@ More info on this at this section: :ref:`providers`.
 SCRAPY_POET_OVERRIDES
 ---------------------
 
+Deprecated. Use ``SCRAPY_POET_RULES`` instead.
+
+SCRAPY_POET_RULES
+-----------------
+
 Default: ``None``
 
 Mapping of overrides for each domain. The format of the such ``dict`` mapping
-depends on the currently set Registry. The default is currently 
-:class:`~.OverridesRegistry`. This can be overriden by the setting below:
-``SCRAPY_POET_OVERRIDES_REGISTRY``.
-
-There are sections dedicated for this at :ref:`intro-tutorial` and :ref:`overrides`.
+depends on the currently set registry. The default is currently 
+:class:`web_poet.rules.RulesRegistry`. This can be overriden by the setting below:
+``SCRAPY_POET_REGISTRY``.
 
+There are sections dedicated for this at :ref:`intro-tutorial` and
+:ref:`rules-from-web-poet`.
 
 SCRAPY_POET_OVERRIDES_REGISTRY
 ------------------------------
 
+Deprecated. Use ``SCRAPY_POET_REGISTRY`` instead.
+
+SCRAPY_POET_REGISTRY
+--------------------
+
 Defaut: ``None``
 
-Sets an alternative Registry to replace the default :class:`~.OverridesRegistry`.
-To use this, set a ``str`` which denotes the absolute object path of the new
-Registry.
+Sets an alternative Registry to replace the default
+:class:`web_poet.rules.RulesRegistry`. To use this, set a ``str`` which denotes
```

Comment:
If I'm not mistaken, like all other Scrapy settings, the class object could also work, not just a str import path.

Reply:
Good call! https://github.com/scrapinghub/scrapy-poet/pull/88/commits/00d5dd6224dfe247e02cdff5f706dfe201128574